benchmarks/benchmark_loop/launch_remote.py

```diff
@@ -0,0 +1,33 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+from coolname import generate_slug
+from sagemaker.pytorch import PyTorch
+
+from syne_tune.backend.sagemaker_backend.sagemaker_utils import get_execution_role
+import syne_tune
+import blackbox_repository
+import benchmarks
+from syne_tune.util import s3_experiment_path
+
+if __name__ == '__main__':
```

Comment:
Are you planning to make this launcher more generic, so we use it to launch all sorts of experiments, for example selecting the SM framework?

Or do you rather think the user should also copy and edit this one?

I'd for sure think users may want to copy and edit benchmark_main.py, but maybe they can also make use of the same launcher?

Reply:
> Are you planning to make this launcher more generic, so we use it to launch all sorts of experiments, for example selecting the SM framework?
> 
> Or do you rather think the user should also copy and edit this one?

I think copy pasting is the best option for now as it lets all options open for the user with minimum amount of code. We could factor a launcher once we have more clarity but I think it is better to leave this one for later as I do not think we have enough clarity on all use-cases yet.

Note: the use-case is slightly different than for remote-launcher that allows to run **one** tuning in the background easily which is a more frequent and delimited task. I still think we should support selecting the framework in the remote-launcher, I can take a look on this one after the holidays.